lib/command.c

```diff
@@ -86,6 +86,8 @@ const char *node_names[] = {
 	"bgp vnc l2",		    // BGP_VNC_L2_GROUP_NODE,
 	"rfp defaults",		    // RFP_DEFAULTS_NODE,
 	"bgp evpn",		    // BGP_EVPN_NODE,
+        "bgp vpn policy ipv4",      // BGP_VPNPOLICY_IPV4_NODE
```

Comment:
I am not sure if the comments are aligned here (GitHub does not display tabs as the project has them configured), but if they are not, please fix it. If they are, please ignore this comment.

Reply:
The comment is aligned, but in any event, this part of the CLI is slated for demolition soon.